src/main/java/com/minecolonies/coremod/entity/ai/citizen/deliveryman/EntityAIWorkDeliveryman.java

```diff
@@ -435,13 +436,13 @@ private boolean hasTools(@NotNull final AbstractBuilding buildingToDeliver)
             return true;
         }
 
-        if (requiredTool.equals(Utils.PICKAXE))
+        /*if (requiredTool.equals(Utils.PICKAXE))
         {
             return InventoryUtils.isPickaxeInItemHandler(new InvWrapper(worker.getInventoryCitizen()),
                     buildingToDeliver.getNeededPickaxeLevel(),
                     buildingToDeliver.getBuildingLevel());
-        }
-        return InventoryUtils.isToolInItemHandler(new InvWrapper(worker.getInventoryCitizen()), requiredTool, buildingToDeliver.getBuildingLevel());
+        }*/
+        return InventoryUtils.isToolInItemHandler(new InvWrapper(worker.getInventoryCitizen()), requiredTool, buildingToDeliver.getNeededPickaxeLevel(), buildingToDeliver.getBuildingLevel());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 191 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)